<launch>
    <node pkg="bot_sim" type="dbscan_bfs_3D" name="dbscan_bfs_3D" output="screen">
        <param name="epsilon" type="double" value="0.2" />
        <param name="minPts" type="int" value="7" />
        <param name="dfs_decrease" type="int" value="7" />
        <param name="dfs_threshold" type="int" value="65" />
        <param name="parent_frame" type="string" value="gimbal_frame" />
        <param name="child_frame" type="string" value="virtual_frame" />
    </node>
</launch>
